project/src/components/Pagination.tsx

```diff
@@ -0,0 +1,26 @@
+const Pagination = () => {
+    return(
+        <>
+        <div className="flex flex-row pt-8 pb-20 gap-12 justify-center items-center">
+            <svg xmlns="http://www.w3.org/2000/svg"viewBox="0 0 32 20"  className="stroke-blue-300 w-6 md:hover:cursor-pointer">
+                <g opacity="0.4">
+                    <path d="M27.6911 8.76277L6.64523 8.76277L11.269 4.13905C11.9067 3.50129 11.9067 2.54466 11.269 1.90691C10.6312 1.26915 9.67457 1.26915 9.03681 1.90691L1.70263 9.24109C1.06487 9.87885 1.06487 10.8355 1.70263 11.4732L9.03681 18.8074C9.67457 19.4452 10.6312 19.4452 11.269 18.8074C11.9067 18.1697 11.9067 17.213 11.269 16.5753L6.64523 11.9515L27.6911 11.9515C28.4883 11.9515 29.2855 11.3138 29.2855 10.3572C29.2855 9.40053 28.4883 8.76277 27.6911 8.76277Z" fill="#458FF6"/>
+                </g>
+            </svg>
```

Comment:
Export this and the following svg into a separate component, you could name them "LeftArrowIcon" and "RightArrowIcon" if you want